### PR TITLE
Update demo environment TLS certificate

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/libero/publisher/issues/349

Similar to https://github.com/libero/environments/pull/49

```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            04:31:02:36:cc:30:f2:52:4a:11:5d:7c:2b:5c:55:69:09:e7
        Signature Algorithm: sha256WithRSAEncryption
        Issuer: C = US, O = Let's Encrypt, CN = Let's Encrypt Authority X3
        Validity
            Not Before: Dec  9 12:22:02 2019 GMT
            Not After : Mar  8 12:22:02 2020 GMT
        Subject: CN = demo.libero.pub
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                RSA Public-Key: (2048 bit)
                Modulus:
                    00:df:35:9d:71:9f:f9:90:90:3a:ad:cb:2e:af:da:
                    d9:d2:5e:19:47:20:8e:1e:52:6e:a4:74:84:ad:27:
                    87:a2:1b:6d:63:16:1f:fc:7d:3c:85:9d:d7:c7:ed:
                    1c:6e:43:fb:2c:cb:b4:79:83:da:41:63:7e:ca:4c:
                    3e:a1:73:76:02:b8:9d:0c:5d:a0:b9:06:6c:12:9e:
                    ae:ed:48:70:e7:d3:cb:ad:a3:39:54:4a:d2:3b:bf:
                    94:87:7d:f3:ce:a8:5c:bb:35:a1:26:aa:ff:e5:b6:
                    17:7b:23:5e:2a:a5:3e:e0:78:93:cd:eb:55:be:a9:
                    c5:a1:d1:67:cc:db:30:45:76:d2:08:89:75:3b:bd:
                    82:ab:18:66:b8:a6:13:0e:76:09:55:ec:ca:58:6e:
                    b3:2e:b6:c2:21:ab:3d:7c:22:ed:8a:d0:59:dc:86:
                    f1:d3:c7:c0:c9:b2:d7:16:8d:15:38:6c:33:c6:1c:
                    1a:0a:ab:77:75:69:47:4a:5b:90:87:81:9f:ab:ee:
                    ac:6c:10:0f:65:55:27:bb:38:e8:74:15:bd:b2:c6:
                    e5:97:32:b7:c2:5e:78:9a:ab:10:65:ac:cc:0a:0c:
                    b4:da:aa:f2:8a:5d:55:f2:4c:5a:f2:d3:38:c3:74:
                    7f:94:6c:a0:82:67:0d:f6:49:aa:b1:1f:0b:ba:a9:
                    03:53
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage: 
                TLS Web Server Authentication, TLS Web Client Authentication
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Subject Key Identifier: 
                AF:71:F7:DE:85:4C:ED:EB:24:75:CD:1F:CF:75:EC:15:22:B9:65:F6
            X509v3 Authority Key Identifier: 
                keyid:A8:4A:6A:63:04:7D:DD:BA:E6:D1:39:B7:A6:45:65:EF:F3:A8:EC:A1

            Authority Information Access: 
                OCSP - URI:http://ocsp.int-x3.letsencrypt.org
                CA Issuers - URI:http://cert.int-x3.letsencrypt.org/

            X509v3 Subject Alternative Name: 
                DNS:demo--api-gateway.libero.pub, DNS:demo--dummy-api.libero.pub, DNS:demo--jats-ingester.libero.pub, DNS:demo--pattern-library.libero.pub, DNS:demo.libero.pub
            X509v3 Certificate Policies: 
                Policy: 2.23.140.1.2.1
                Policy: 1.3.6.1.4.1.44947.1.1.1
                  CPS: http://cps.letsencrypt.org

            CT Precertificate SCTs: 
                Signed Certificate Timestamp:
                    Version   : v1 (0x0)
                    Log ID    : F0:95:A4:59:F2:00:D1:82:40:10:2D:2F:93:88:8E:AD:
                                4B:FE:1D:47:E3:99:E1:D0:34:A6:B0:A8:AA:8E:B2:73
                    Timestamp : Dec  9 13:22:02.192 2019 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                30:44:02:20:28:B7:CB:93:1F:0C:27:6C:92:EE:36:65:
                                E1:81:B6:55:DD:95:22:5B:E9:AB:DA:FB:38:F3:FE:A5:
                                30:3C:DE:53:02:20:2A:D5:BE:0B:2F:96:F1:81:3B:4E:
                                0B:AD:03:F1:93:9C:C3:DC:E8:3C:0E:2A:6F:AE:39:43:
                                46:04:27:31:5A:4A
                Signed Certificate Timestamp:
                    Version   : v1 (0x0)
                    Log ID    : 07:B7:5C:1B:E5:7D:68:FF:F1:B0:C6:1D:23:15:C7:BA:
                                E6:57:7C:57:94:B7:6A:EE:BC:61:3A:1A:69:D3:A2:1C
                    Timestamp : Dec  9 13:22:02.228 2019 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                30:46:02:21:00:A0:8A:88:44:03:1F:D1:E8:68:33:07:
                                4B:AD:21:C5:ED:39:08:70:32:28:8F:10:6D:F9:3B:3B:
                                A6:50:E0:B5:EE:02:21:00:FA:AC:F7:F6:FD:BC:85:A6:
                                B5:94:12:F1:75:77:51:8B:47:42:E6:29:ED:15:F0:8B:
                                35:C9:E3:37:DB:CA:B6:09
    Signature Algorithm: sha256WithRSAEncryption
         2a:f0:c2:be:4d:c5:9b:28:ec:6c:97:d7:d3:5f:61:03:94:a9:
         5c:c3:76:dd:bb:80:c5:26:c3:09:de:3b:c5:55:03:97:08:c1:
         a0:62:70:10:99:06:4d:a3:6c:6a:13:e6:3b:87:ce:5b:98:d2:
         16:b1:e9:b6:2e:b9:fa:73:79:8d:27:38:a8:65:e2:6a:4b:93:
         5b:d0:f8:c0:3f:27:27:6a:ba:c8:c4:f9:de:f0:57:13:0c:8e:
         cd:6e:5f:8f:66:4f:2b:9c:27:0e:da:a4:85:e7:00:1f:43:46:
         56:2d:33:5c:df:68:19:16:0a:d1:c5:f4:8c:0c:68:d8:ba:7c:
         13:57:6b:59:09:57:4c:7c:b9:99:a1:9e:46:f9:87:30:ed:b1:
         c4:f8:13:c3:e5:1c:10:22:42:8c:f1:6c:5e:fb:2a:c0:ea:f6:
         28:78:b5:5f:66:73:2c:f1:bf:0f:54:6d:6e:83:28:ae:ab:da:
         17:60:0d:e1:7e:80:cb:cb:e8:a4:85:3b:05:77:6b:8a:0e:df:
         cd:71:43:02:1a:e4:e2:ac:62:82:4b:50:1e:60:98:45:66:74:
         bb:12:d1:e0:07:c5:47:2e:9d:4a:38:68:c1:b2:04:bf:2a:16:
         98:e9:8b:d7:a9:f7:0e:10:9e:a6:a4:6f:25:b1:0e:f3:75:8f:
         29:dd:72:a4
-----BEGIN CERTIFICATE-----
MIIF1TCCBL2gAwIBAgISBDECNsww8lJKEV18K1xVaQnnMA0GCSqGSIb3DQEBCwUA
...
-----END CERTIFICATE-----
```

### CODE REVIEW

**Comment by giorgiosironi** [comment]
In order to be deployed, this will need not just to be merged, but also git-tagged.

**Comment by giorgiosironi** [comment]
Giving https://github.com/orgs/libero/teams/elife-developers `Write` permissions here since merging requires reviews and passing tests anyway.

File: files/demo/https_certificate.pem [hunk 1]
```diff
@@ -60,3 +60,4 @@ X4Po1QYz+3dszkDqMp4fklxBwXRsW10KXzPMTZ+sOPAveyxindmjkW8lGy+QsRlG
 PfZ+G6Z6h7mjem0Y+iWlkYcV4PIWL1iwBi8saCbGS5jN2p8M+X+Q7UNKEkROb3N6
 KOqkqm57TH2H3eDJAkSnh6/DNFu0Qg==
 -----END CERTIFICATE-----
+
```
Comment (davidcmoulton) [inline files/demo/https_certificate.pem:63]: Any danger of this new line at the end causing a problem?

Comment (giorgiosironi) [reply files/demo/https_certificate.pem:63]: Would normally think the `END CERTIFICATE` armoring should make this robust. To cross-check, the `unstable` environment has now [got the additional newline](https://github.com/libero/environments/blob/master/files/unstable/https_certificate.pem#L63) and is [still working](https://unstable.libero.pub/).

**Review by davidcmoulton — APPROVED** [review]

**Comment by giorgiosironi** [comment]
Also slightly relaxing https://github.com/libero/environments/settings/branch_protection_rules/4696784 to allow `Write` access users to approve.

**Comment by giorgiosironi** [comment]
https://demo.libero.pub/ being served successfully with new certificate.

**Pull request merged**